### PR TITLE
Fix blank page in creative on some rare case

### PR DESCRIPTION
**Pull request by kilbith**
When you click on previous from page 1 and if the number of items is a multiple of 24 (= inventory size), then `start_i` is equal to `inv_size` (thus with no rest), resulting in a blank page.

### CODE REVIEW

File: mods/creative/init.lua [hunk 1]
```diff
@@ -163,7 +163,7 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	local formspec = player:get_inventory_formspec()
 	local filter = formspec:match("filter;;([%w_:]+)") or ""
 	local start_i = formspec:match("list%[detached:creative_".. player_name ..";.*;(%d+)%]")
-	local tab_id = tonumber(formspec:match("tabheader%[.*;(%d+)%;.*%]"))
+	local tab_id = tonumber(formspec:match("tabheader%[.*;(%d)%;.*%]"))
```
Comment (Ekdohibs) [inline mods/creative/init.lua]: Why was this changed?

**Comment by Ekdohibs** [comment]
Looks good!

File: mods/creative/init.lua (hunk 1 above)
Comment (kilbith) [inline mods/creative/init.lua]: To limit the matching from 0 to 9, and not 0 to infinite (not needed for tab indexes).

Comment (Ekdohibs) [inline mods/creative/init.lua]: Yes, but if the number of tabs in increased later, it is best to keep the current code.

Comment (kilbith) [inline mods/creative/init.lua]: OK, will change back.

**Comment by Fixer-007** [comment]
Could you also please align "1 / XX" numbers between < and > in the middle? Now it looks little displaced to upper left near <

**Comment by paramat** [comment]
:+1:

**Comment by kilbith** [comment]
Pushed a little design change to keep the crafting tab size stationary, following @paramat's suggestion.

**Comment by paramat** [comment]
Ah cool i was going to open an issue about that, thanks.

**Comment by paramat** [comment]
250be40ecd1c736c18f5428094561cd74dff4177
